lib/generators/curation_concerns/install_generator.rb

```diff
@@ -101,8 +101,11 @@ def resource_types_config
       copy_file "config/authorities/resource_types.yml", "config/authorities/resource_types.yml"
     end
 
-    def simple_form_bootstrap
-      generate 'simple_form:install --bootstrap'
```

Comment:
You could keep this generator in here, then copy over the generated files. This ensures if later versions of simple form install additional files, they will be generated.